cypress/pageobject/Login/LoginPage.ts

```diff
@@ -1,6 +1,9 @@
-// LoginPage.ts
+ // LoginPage.ts
 
 class LoginPage {
+  submitButtonSelector = "#login";
+  languageSelector = "#language-selector";
+  sidebarSelector = "#sidebar";
```

Comment:
Hmm, neither language or sidebar selectors are present in Login.tsx; You have to add them. Have you tested this locally?